spec/sidekiq/throttled/fetch_spec.rb

```diff
@@ -5,7 +5,7 @@
 
 require "support/working_class_hero"
 
-RSpec.describe Sidekiq::Throttled::Fetch, :sidekiq => :disabled do
+RSpec.describe Sidekiq::Throttled::Fetch, :sidekiq => :disabled, :verify_stubs => false do
```

Comment:
This was needed, because in the newest sidekiq, redis method calls are passed via some complicated sort of delegation (https://github.com/sidekiq/sidekiq/blob/main/lib/sidekiq/redis_client_adapter.rb), and `expect` rspec matcher complains about not implemented method on that object.